analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -638,7 +638,7 @@ def parse_convert_reports(input_dirs: List[str],
 
     number_of_reports = len(all_reports)
     if out_format == "codeclimate":
-        return codeclimate.convert(all_reports), number_of_reports
+        return codeclimate.convert(all_reports, severity_map), number_of_reports
```

Comment:
Pylint complains that this line is too long:
```
codechecker_analyzer/cmd/parse.py:641:80: E501 line too long (80 > 79 characters)
make[1]: *** [pycodestyle] Error 1
```
Could you please fix this problem?

Reply:
fixed